src/components/Filter.js

```diff
@@ -1,9 +1,15 @@
 import React from "react";
 
-function Filter({ onCategoryChange }) {
+function Filter({ onCategoryChange, onSearchChange, search }) {
   return (
     <div className="Filter">
-      <input type="text" name="search" placeholder="Search..." />
+      <input
+      type="text"
+      name="search"
+      value={search}
+      placeholder="Search..."
+      onChange={onSearchChange}
+      />
```

Comment:
```suggestion
      <input
        type="text"
        name="search"
        value={search}
        placeholder="Search..."
        onChange={onSearchChange}
      />
```

Reply:
Just a warning I continued to be terrible at indenting my component props until you showed me the right way on like Friday or Thursday so apologies in advance but I promise I have gotten better lol.

Reply:
Haha, I see, no worries!